### PR TITLE
Add logos to homepage

### PR DESCRIPTION
**Pull request by francisschmaltz**
- Add logos of Sourcegraph customers to homepage
screen shot 2019-01-04 at 4 06 08 pm
screen shot 2019-01-04 at 4 06 27 pm

- Update search bar on modal to be sticky to top of page
screen shot 2019-01-04 at 4 06 49 pm
screen shot 2019-01-04 at 4 06 38 pm

- Update buttons to be full width on small mobile devices
screen shot 2019-01-04 at 4 06 15 pm

### CODE REVIEW

File: web/src/search/input/MainPage.scss [hunk 1]
```diff
@@ -316,6 +376,10 @@ body.main-page {
         position: -webkit-sticky;
         z-index: 2;
         box-shadow: rgba(0, 0, 0, 0.35) 0 4rem 8rem 12rem;
+		// stylelint-disable-next-line declaration-block-no-duplicate-properties
+		&:nth-of-type(2) {
```
Comment (dadlerj) [inline web/src/search/input/MainPage.scss]: Appears as though there's a tab/space issue here...

File: web/src/search/input/MainPage.tsx [hunk 2]
```diff
@@ -333,6 +333,27 @@ export class MainPage extends React.Component<Props, State> {
                     </div>
                 </section>
                 <div className="about-section-container">
+					<section className="company-logos">
+						<div className="container">
+							<div className="row">
+								<div className="col-12">
+									<h2>Companies developing with Sourcegraph</h2>
```
Comment (dadlerj) [inline web/src/search/input/MainPage.tsx]: Like we chatted about offline, would love to center this on the devs themselves — e.g. like "Powering developers at..."

File: web/src/search/input/MainPage.tsx [hunk 3]
```diff
@@ -333,6 +333,27 @@ export class MainPage extends React.Component<Props, State> {
...
+									</div>
+									<div className="col-3 logo">
+										<figure className="logo-image logo2"/>
+									</div>
+									<div className="col-3 logo">
+										<figure className="logo-image logo3"/>
+									</div>
+									<div className="col-3 logo">
+										<figure className="logo-image logo4"/>
+									</div>
```
Comment (dadlerj) [inline web/src/search/input/MainPage.tsx]: And would love to add a sentence at the bottom that alludes to all of the other customers and free/oss users that restrict our logo rights, e.g. if we used the text I gave an example of above, something like  "...and thousands more around the world" (or something)

Comment (francisschmaltz) [reply web/src/search/input/MainPage.tsx]: I like adding the copy "Powering developers at" but I don't think having the ellipsis before and after looks good. When we have logos it also implies that there are other people using it outside of what's listed.

Comment (dadlerj) [reply web/src/search/input/MainPage.tsx]: Sure

**Comment by francisschmaltz** [comment]
@dadlerj Updated PR and images to reflect current state and real customer logos.

File: web/src/search/input/MainPage.tsx (hunk 2 above)
Comment (francisschmaltz) [reply web/src/search/input/MainPage.tsx]: Added!

File: web/src/search/input/MainPage.scss (hunk 1 above)
Comment (francisschmaltz) [reply web/src/search/input/MainPage.scss]: Resolved

**Pull request merged**